keepassxc-browser/background/page.js

```diff
@@ -317,7 +317,7 @@ page.updateContextMenu = async function(tab, credentials) {
             // Show username inside [] if there are KPH attributes inside multiple credentials
             const attributeName = Object.keys(attribute)[0].slice(5);
             const finalName = credentials.length > 1
-                ? `[${cred.login}] ${attributeName}`
+                ? `[${cred?.login}] ${attributeName} (${cred?.name})`
```

Comment:
Wouldn't this add empty parenthesis at the end if cred.name is empty? Is that the title of the entry?

Reply:
That's the title yes. I think I'm gonna tune this a little bit, making the title separate, and maybe using the UUID as the internal ID for the menuitem if that's possible.

Reply:
Made the fix. Now UUID is used for the menuitem ID, and the title of menuitem includes the entry title or just pure index if there is no title at all.